prune/prunables.go

```diff
@@ -0,0 +1,104 @@
+// Copyright 2021 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package prune
+
+import (
+	"fmt"
+
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	batchv1 "k8s.io/api/batch/v1"
+	corev1 "k8s.io/api/core/v1"
+)
+
+// Some default pruneable functions
+
+// DefaultPodIsPruneable is a default IsPruneableFunc to be used specifically with Pod resources.
+// This can be overridden by registering your own IsPruneableFunc via the RegisterIsPruneableFunc method
+func DefaultPodIsPruneable(obj client.Object) error {
+	gvk := schema.GroupVersionKind{
+		Group:   "core",
+		Version: "v1",
+		Kind:    "Pod",
+	}
+
+	// if the object is not a Pod then it is not pruneable
+	if obj.GetObjectKind().GroupVersionKind() != gvk {
+		return fmt.Errorf("can not prune object as it is not a Pod")
+	}
+
+	// convert to a Pod object so we can do Pod specific checks
+	pod := &corev1.Pod{}
+	err := ConvertClientObjectToKind(obj, pod)
+
+	if err != nil {
```

Comment:
nit: drop the space between 46 and 48.

Reply:
fixed in https://github.com/operator-framework/operator-lib/pull/105/commits/1fca33a0903e9f38f9f39390c315c8618bbc2414